Documentation/core-repos.md

```diff
@@ -1,43 +1,80 @@
-* [dotnet/vblang](https://github.com/dotnet/vblang) - VB spec
-* [dotnet/fsharp](https://github.com/dotnet/fsharp) - The F# compiler, FSharp.Core library, and tools for F#
+|Repository                                                         |Description                                  |
+|-------------------------------------------------------------------|---------------------------------------------|
+|[aspnet/AspNetCore](https://github.com/aspnet/AspNetCore)          |ASP.NET Core framework                       |
+|[aspnet/AspNetCore.Docs](https://github.com/aspnet/AspNetCore.Docs)|ASP.NET Core documentation                   |
+|[aspnet/Scaffolding](https://github.com/aspnet/scaffolding)        |ASP.NET Core code generators                 |
+|[aspnet/WebSdk](https://github.com/aspnet/websdk)                  |MSBuild tasks & targets for ASP.NET Core apps|
 
-## NuGet
+## Build platform
 
-* [nuget/home](https://github.com/nuget/home) - NuGet client
+|Repository                                               |Description                            |
+|---------------------------------------------------------|---------------------------------------|
+|[microsoft/msbuild](https://github.com/microsoft/msbuild)|Build platform for .NET & Visual Studio|
 
-## ASP.NET Core
+## Entity Framework and data access
 
-* [aspnet/home](https://github.com/aspnet/home) - ASP.NET Core framework
-* [aspnet/Docs](https://github.com/aspnet/Docs) - ASP.NET and ASP.NET Core documentation
+|Repository                                                                 |Description                           |
+|---------------------------------------------------------------------------|--------------------------------------|
+|[aspnet/EntityFrameworkCore](https://github.com/aspnet/EntityFrameworkCore)|EF Core                               |
+|[aspnet/EntityFramework6](https://github.com/aspnet/EntityFramework6)      |EF 6                                  |
+|[aspnet/EntityFramework.Docs](https://github.com/aspnet/AspNetCore.Docs)   |EF Core & EF 6 documentation          |
+|[dotnet/sqlclient](https://github.com/dotnet/sqlclient)                    |SqlClient data provider for SQL Server|
+
+## Languages
+
+|Repository                                               |Description     |
+|---------------------------------------------------------|----------------|
+|[dotnet/roslyn](https://github.com/dotnet/roslyn)        |Roslyn compiler |
+|[dotnet/csharplang](https://github.com/dotnet/csharplang)|C# specification|
+|[dotnet/vblang](https://github.com/dotnet/vblang)        |VB specification|
+|[dotnet/fsharp](https://github.com/dotnet/fsharp)        |The F# compiler, FSharp.Core library, and tools for F#|
+
+## NuGet
+
+|Repository                                                  |Description |
+|------------------------------------------------------------|------------|
+|[nuget/home](https://github.com/nuget/home)                 |NuGet client|
```

Comment:
Isn't `Nuget.Client` at https://github.com/NuGet/NuGet.Client/ ?

Reply:
Yes, but issues are disabled at NuGet/NuGet.Client. The repo tells folks to open issues at NuGet/home.